massdash/loaders/GenericChromatogramLoader.py

```diff
@@ -3,42 +3,37 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 """
 
-from abc import abstractmethod
+from abc import abstractmethod, ABCMeta
 import pandas as pd 
 from typing import Dict, List, Union, Literal
 
 # Loader
-from .GenericLoader import GenericLoader
+from .GenericRawDataLoader import GenericRawDataLoader
 # Structs
 from ..structs import TransitionGroup
 
-class GenericChromatogramLoader(GenericLoader):
+class GenericChromatogramLoader(GenericRawDataLoader, metaclass=ABCMeta):
     '''
     Abstract class for loading raw chromatogram data
     
     Attributes:
         rsltsFile: (str) The path to the report file (DIANN-TSV or OSW)
         dataFiles: (str/List[str]) The path to the mzML file(s)
-        rsltsFileType: (str) The type of results file (OpenSWATH or DIA-NN)
         verbose: (bool) Whether to print debug messages
         mode: (str) Whether to run in module or GUI mode
     '''
 
-    def __init__(self, 
-                 rsltsFile: str, 
-                 dataFiles: Union[str, List[str]], 
-                 rsltsFileType: Literal['OpenSWATH', 'DIA-NN'] = 'DIA-NN', 
-                 verbose: bool=False, 
-                 mode: Literal['module', 'gui'] = 'module'):
-        super().__init__(rsltsFile, dataFiles, None, rsltsFileType, verbose, mode)
+    def __init__(self, **kwargs): 
+        super().__init__(**kwargs)
```

Comment:
The idea was to not have to copy the arguments all the way up the chain (e.g. if ResultsLoader is edited then we do not have to edit GenericChromatogramLoader)
It definitely is harder to read though so I'm not sure

Reply:
mm, that makes sense. I think it's good as is, it will make maintaining easier. I would add a doc-string link to where the args are laid out though, so it's easier to track references/inheritance.